pool/src/pool.rs

```diff
@@ -427,6 +427,47 @@ impl Pool {
 		tx_buckets
 	}
 
+	/// Get a list of transaction that are potentially evictable, i.e. only transaction with no children
+	pub fn get_evictable_transactions(&self) -> Vec<Transaction> {
```

Comment:
Why not rely on `bucket_transactions`? Doesn't seem a lot more costly and would avoid all that duplication.